library/examples/hate_speech.py

```diff
@@ -37,7 +37,6 @@
     preferred_load_origin=LoadOrigin.local,
     pretrained_model="distilbert-base-uncased",
     user_name="semy",
-    repo_name="finetuning-tweeteval-hate-speech",
```

Comment:
Naming happens through ``{user_name}/{model.id}``, this was left behind.

Reply:
so that means I can't load an already finetuned model (like yours) in my own pipeline, since the names will not guaranteed to match?